TravelAgency-service/src/main/java/cz/muni/fi/travelAgency/facade/ExcursionFacadeImpl.java

```diff
@@ -25,17 +28,21 @@
     @Autowired
     private ExcursionService excursionService;
 
+    @Autowired
+    private TripService tripService;
+
     /**
      * Mapper responsible for mapping DTOs to Entities.
      */
     @Autowired
     private BeanMappingService beanMappingService;
 
     @Override
-    public void createExcursion(ExcursionDTO excursionDTO) {
+    public Long createExcursion(ExcursionCreateDTO excursionDTO) {
```

Comment:
This is not wrong approach but we should be consistent. Why did you chose this approach?

Reply:
Because ExcursionCreateDTO stores id of trip, ExcursionDTO store whole object.

Reply:
Ok, I think we should reconsider using the DTOs in this or the other manner.